packages/relayer/http/routes.go

```diff
@@ -0,0 +1,5 @@
+package http
+
+func (srv *Server) configureRoutes() {
+	srv.echo.GET("/health", srv.Health)
```

Comment:
Looks like `/healthz` is a more commonly used health checking path? ( not quite sure )